scripts/release/brew-cask.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+# Copyright (C) 2016 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+if [ $# -eq 0 ]; then
+	echo "usage: $0 <homebrew_dir> <release_version>"
+	exit
+fi
+
+HOMEBREW_REPO=$1
+RELEASE_VERSION=$2
+
+cd $HOMEBREW_REPO/Casks
+git checkout -b minishift-$RELEASE_VERSION
+SHA_ID=$(curl -sL https://github.com/minishift/minishift/releases/download/v$RELEASE_VERSION/minishift-$RELEASE_VERSION-darwin-amd64.tgz.sha256)
+
+if [[ $SHA_ID = "Not Found" ]]; then
+	echo "version" $RELEASE_VERSION "not found on github"
+	exit
+fi
+
+CHECKPOINT=$(curl -s https://github.com/minishift/minishift/releases.atom | shasum -a 256 | cut -d' ' -f1)
+
+sed -i "" "s|version '.*'|version '$RELEASE_VERSION'|" minishift.rb
+sed -i "" "s|sha256 '.*'|sha256 '$SHA_ID'|" minishift.rb
+sed -i "" "s|checkpoint: '.*'|checkpoint: '$CHECKPOINT'|" minishift.rb
+
+brew cask install minishift.rb
+if [ $? -ne 0 ]; then
+       echo "Install failed"
+       exit
+fi       
+
+brew cask audit minishift.rb --download
+if [ $? -ne 0 ]; then
+       echo "Audit failed"
+       exit
+fi       
+
+brew cask style minishift.rb
+if [ $? -ne 0 ]; then
+       echo "Style check failed"
+       exit
+fi       
+
+git add minishift.rb
+git commit -m "Update minishift to $RELEASE_VERSION"
+
+echo -e "\nImportant-----> Push the changes to your local repo"
```

Comment:
Since user is at minishift repo after the end of this script, we need to add message to go to your repo and send pull request.